### ISSUE TITLE
Remove extra newlines in returned text

### ISSUE THREAD

**Issue by ehandal** [microsoft/pyright#1712]
I'm using coc-pyright and seeing text like this returned on textDocument/hover:
```
"contents": {
    "kind": "markdown",
    "value": "```python\n(variable) x: Literal[0]\n```\n"
},
```
and completionItem/resolve:
```
"documentation": {
    "kind": "markdown",
    "value": "```python\nclass ValueError()\n```\n"
}
```

This causes coc-pyright to show two line results on hover/completion like this:

![image](https://user-images.githubusercontent.com/23243879/113359563-a1013180-930d-11eb-9d9e-32344c33db60.png)

Not sure if coc-pyright should be expected to handle the newlines or strip them to properly display these on one line. If so, I could submit an issue there instead.

Using Pyright 1.1.126

**Comment by jakebailey** [comment]
VS Code does strip, but we should probably stick a `trimEnd` here: https://github.com/microsoft/pyright/blob/master/packages/pyright-internal/src/languageService/hoverProvider.ts#L448

We have this already for the end of docstrings, but are missing it for hover/completion when there's no docstring after.

**Comment by erictraut** [comment]
Thanks for reporting the problem.

Since this isn't related to core type checker functionality, I'm going to transfer this bug to the pylance-release repo so it gets the appropriate attention & discussion.
